### PR TITLE
Testing models and controllers

### PR DESCRIPTION
**Pull request by alaxalves**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
This PR depends on https://github.com/publiclab/spectral-workbench/pull/499, since there are some ruby syntax changes not accepted by current ruby version.

**Comment by jywarren** [comment]
Hmm was this already in 499? All I see are indentation changes, I think! Or maybe I'm misreading?

Thanks Alax!

File: test/unit/spectrum_test.rb [hunk 1]
```diff
@@ -164,20 +164,20 @@ class SpectrumTest < ActiveSupport::TestCase
   test "compare" do
   	spectrum1 = Spectrum.first
   	spectrum2 = Spectrum.second
```
Comment (cesswairimu) [inline test/unit/spectrum_test.rb:166]: Hello @alaxalves great job fixing the indentation...maybe we could also fix it on these lines too also at line 176...Thanks

**Comment by alaxalves** [comment]
@cesswairimu @jywarren Actually those changes have already been on #499 :laughing: :laughing: :laughing: I'll close this for now

**Pull request closed without merging**